doom-modeline-segments.el

```diff
@@ -771,12 +771,12 @@ level."
                 ('finished  (if flycheck-current-errors
                                 (let-alist (doom-modeline--flycheck-count-errors)
                                   (doom-modeline-checker-icon
-                                   "block" "🚫" "!"
+                                   "error_outline" "🚫" "!"
                                    (cond ((> .error 0) 'doom-modeline-urgent)
                                          ((> .warning 0) 'doom-modeline-warning)
                                          (t 'doom-modeline-info))))
                               (doom-modeline-checker-icon "check" "✓" "-" 'doom-modeline-info)))
-                ('running     (doom-modeline-checker-icon "access_time" "⏱" "*" 'doom-modeline-debug))
+                ('running     (doom-modeline-checker-icon "update" "⏳" "*" 'doom-modeline-debug))
```

Comment:
Looks the icon is different from the screenshot. Should it be `hourglass_empty`?

Reply:
Ah yep, that's a mistake and should indeed be `hourglass_empty`. I think `hourglass_bottom` would be ideal, but that doesn't seem to have made it's way through the various levels of releases yet.